pypowsybl/voltage_initializer.py

```diff
@@ -0,0 +1,137 @@
+from typing import Dict, List, Tuple
```

Comment:
it would be better for now (before refactoring) to import with a _ to hide this import for users
I mean 
```python
from typing import Dict as _Dict, List as _List, Tuple as _Tuple
```

Reply:
now that the refactoring is merged it should be good to do the same

Reply:
Done